openmc/deplete/openmc_operator.py

```diff
@@ -223,8 +223,9 @@ def _get_burnable_mats(self):
             if mat.depletable:
                 burnable_mats.add(str(mat.id))
                 if mat.volume is None:
-                    raise RuntimeError("Volume not specified for depletable "
-                                       "material with ID={}.".format(mat.id))
+                    msh = ("Volume not specified for depletable material with "
+                           f"ID={mat.id} Name={mat.name}.")
```

Comment:
When the name is empty, this will look a bit weird (`...material with ID=10 Name=.`) Can you change so that we don't display name in that case?

Reply:
I put in a commit where the msg is extended if name is not but I could do this if you prefer.
```python
if mat.name is None:
    msg = ("Volume not specified for depletable material "
           f"with ID={mat.id}.")
else:       
    msg = ("Volume not specified for depletable material "
           f"with ID={mat.id} Name={mat.name}.")
raise RuntimeError(msg)
```

Reply:
Yeah, let's go with this approach. Thanks!